src/server/connectors/address-lookup/organisationRemovaFromLookup.test.js

```diff
@@ -0,0 +1,303 @@
+const {
```

Comment:
Minor point but filename is missing an "l" in "Removal" which makes in in wrong order alphabetically